_tests/browser_tests/README.md

```diff
@@ -0,0 +1,92 @@
+## Quick start:
+
+_Note: If you haven't, make sure to run `./setup.sh from the project root` to install Protractor globally._
+
+From the project root:
+
+```sh
+# Update Selenium Server binaries:
+webdriver-manager update
+```
+
+From this directory:
+
+```sh
+# Start Selenium Server:
+webdriver-manager start
+
+# Open a new tab and tell protractor to start testing:
+protractor conf.js
+```
+
+If you want to test a server other than your local instance (assuming port 7000 is your local CFGov server) use the `--baseUrl` flag. Ex:
+
+```sh
+protractor conf.js --baseUrl=http://beta.consumerfinance.gov
```

Comment:
Since the port is mentioned on L:22 perhaps add a port number example here. I run on 7001 so took a moment to realize `protractor conf.js --baseUrl=http://localhost:7001` was needed.